contracts/strategies/aura/base/AuraStrategyAuraBAL.sol

```diff
@@ -0,0 +1,725 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity 0.5.16;
+pragma experimental ABIEncoderV2;
+
+import "hardhat/console.sol";
```

Comment:
Last little thing :) Please remove

Reply:
Thank you for the reminder, just removed it.